src/cloud/digitalocean/provisioner.ts

```diff
@@ -45,22 +45,26 @@ class Provisioner {
    * One-click setup of a VM
    * See freedom-module.json for return and error types
    * @param {String} name of VM to create
+   * @param {String} region to create VM in
    * @return {Promise.<Object>}
    */
-  public start = (name: string) : Promise<Object> => {
+  public start = (name: string, region?: string) : Promise<Object> => {
     this.sendStatus_("START");
     // Do oAuth
     return this.doOAuth_().then((oauthObj: any) => {
       this.state_.oauth = oauthObj;
       return this.getSshKey_(name);
-    // Get SSH keys
+      // Get SSH keys
     }).then((keys: KeyPair) => {
       this.state_.ssh = keys;
-      return this.setupDigitalOcean_(name);
-    // Setup Digital Ocean (SSH key + droplet)
+      if (typeof region === "undefined") {
+        region = "nyc3";
```

Comment:
Function parameters can have defaults, i.e.
```typescript
public start = (name: string, region: string = 'nyc3'): Promise<Object> => {
  // do stuff
}
```
You could also extract `nyc3` as a `const`.